app/src/webview/java/org/mozilla/focus/webview/TrackingProtectionWebViewClient.java

```diff
@@ -9,17 +9,14 @@
 import android.net.Uri;
 import android.os.AsyncTask;
 import android.support.annotation.WorkerThread;
-import android.webkit.WebResourceRequest;
-import android.webkit.WebResourceResponse;
-import android.webkit.WebView;
-import android.webkit.WebViewClient;
-
+import android.webkit.*;
```

Comment:
\[Checkstyle\] ERROR: Using the '\.\*' form of import should be avoided \- android\.webkit\.\*\.